SpatialGDK/Source/SpatialGDK/Public/Interop/Connection/SpatialWorkerConnection.h

```diff
@@ -83,20 +89,21 @@ class SPATIALGDK_API USpatialWorkerConnection : public UObject, public FRunnable
 	UGlobalStateManager* GlobalStateManager;
 
 private:
-	void ConnectToReceptionist(bool bConnectAsClient);
-	void ConnectToLocator();
+	void ConnectToReceptionist(bool bConnectAsClient, uint32 PlayInEditorID);
```

Comment:
nit - ID is normally Id I think. It's special.

Reply:
Discussed. NetDriver has it as `ID`, so keeping it like that for consistency.